admin/server-configuration.rst

```diff
@@ -102,7 +102,7 @@ username              Onyx
 password              onyx-demo
 ===================== ===========
 
-Your organization may have already changed the defaults. You can check the values in the *onyx-config.properties* file ( `What You Need from Your organization <installation.html#what-you-need-from-your-organization>`_ ). If the settings are different from the defaults shown in the table , note them so you can use them when Creating a
+Your organization may have already changed the defaults. You can check the values in the *onyx-config.properties* file ( :ref:`What You Need from Your organization <what-you-need-from-your-organization>`). If the settings are different from the defaults shown in the table , note them so you can use them when Creating a
```

Comment:
That's not the way of using :ref:

http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-arbitrary-locations